lib/process-services/src/lib/process-list/components/start-process.component.scss

```diff
@@ -1,54 +1,61 @@
-.adf {
-    &-start-process {
-        width: 96%;
-        margin-left: auto;
-        margin-right: auto;
-        margin-top: 10px;
-        .mat-select-trigger {
-            font-size: 14px !important;
-        }
-        mat-form-field {
-            width: 100%;
-        }
-        mat-select {
-            width: 100%;
-            padding: 16px 0 0;
-        }
-        .mat-content-actions {
-            text-align: right;
-        }
-        .mat-button {
-            text-transform: uppercase !important;
-        }
-    }
-    &-process-input-container {
-        mat-form-field {
-            width: 100%;
+@mixin adf-process-services-create-theme($theme) {
```

Comment:
If you change the style to mixin, it should be called where other mixins are, and also component should not reference it in the styles attribute.

Reply:
check the _index.scss for examples

Reply:
sorry forgot to commit the file for import

Reply:
also, the .ts file should not contain the following line now:
```
styleUrls: ['./start-form.component.scss'],
```